SalesforceSDK/Core/Auth/AccountManager.cs

```diff
@@ -166,18 +169,20 @@ public static void SwitchAccount()
         /// </summary>
         /// <param name="loginOptions"></param>
         /// <param name="authResponse"></param>
-        public static async Task<Account> CreateNewAccount(LoginOptions loginOptions, AuthResponse authResponse)
+        /// <param name="cancellationToken"></param>
+        public static async Task<Account> CreateNewAccount(LoginOptions loginOptions, AuthResponse authResponse,
```

Comment:
I thought we already renamed all the methods. I think a new pull request for that should be good.